build.gradle

```diff
@@ -160,10 +160,6 @@ task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updat
             into(folder)
         }
     }
-    from("${assetsDirectory}/icons") {
-        into('icons')
-        exclude("icon.icns")
-    }
```

Comment:
We can likely remove `all_platforms`, some details on the forums: https://forums.triplea-game.org/topic/168/download-counts-of-game-installers/5

We have not come to conclusion in that thread, it seems the genesis of all_platforms was to workaround problems with the previous windows install builder. If we choose to support all_platforms, we should add it to our download screen and begin testing with it in addition to other OS installs. My vote personally is to axe-it so we can reduce the overhead and focus on the core installers. Additional reason, there have been no issues opened reporting a problem with the current OS specific installers that required the `all_platforms` zip.